pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -0,0 +1,279 @@
+func (o *ReconcileClusterRoleBindingsOptions) RunReconcileClusterRoleBindings(cmd *cobra.Command, f *clientcmd.Factory) error {
+	changedClusterRoleBindings, err := o.ChangedClusterRoleBindings()
+	if err != nil {
+		return err
+	}
+
+	if len(changedClusterRoleBindings) == 0 {
+		return nil
+	}
+
+	if (len(o.Output) != 0) && !o.Confirmed {
+		list := &kapi.List{}
+		for _, item := range changedClusterRoleBindings {
+			list.Items = append(list.Items, item)
+		}
+
+		if err := f.Factory.PrintObject(cmd, list, o.Out); err != nil {
+			return err
+		}
+	}
+
+	if o.Confirmed {
+		return o.ReplaceChangedRoleBindings(changedClusterRoleBindings)
+	}
+
+	return nil
+}
+
+// ChangedClusterRoleBindings returns the role bindings that must be created and/or updated to
+// match the recommended bootstrap policy
+func (o *ReconcileClusterRoleBindingsOptions) ChangedClusterRoleBindings() ([]*authorizationapi.ClusterRoleBinding, error) {
+	changedRoleBindings := []*authorizationapi.ClusterRoleBinding{}
+
+	bootstrapClusterRoleBindings := bootstrappolicy.GetBootstrapClusterRoleBindings()
+	for i := range bootstrapClusterRoleBindings {
+		expectedClusterRoleBinding := &bootstrapClusterRoleBindings[i]
+
+		actualClusterRoleBinding, err := o.RoleBindingClient.Get(expectedClusterRoleBinding.Name)
+		if kapierrors.IsNotFound(err) {
+			changedRoleBindings = append(changedRoleBindings, expectedClusterRoleBinding)
+			continue
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		if updatedClusterRoleBinding, needsUpdating := computeUpdatedBinding(expectedClusterRoleBinding, actualClusterRoleBinding, o.Union); needsUpdating {
+			changedRoleBindings = append(changedRoleBindings, updatedClusterRoleBinding)
+		}
+	}
+
+	return changedRoleBindings, nil
+}
+
+// ReplaceChangedRoleBindings will reconcile all the changed system role bindings back to the recommended bootstrap policy
+func (o *ReconcileClusterRoleBindingsOptions) ReplaceChangedRoleBindings(changedRoleBindings []*authorizationapi.ClusterRoleBinding) error {
+	for i := range changedRoleBindings {
+		roleBinding, err := o.RoleBindingClient.Get(changedRoleBindings[i].Name)
+		if err != nil && !kapierrors.IsNotFound(err) {
+			return err
+		}
+
+		if kapierrors.IsNotFound(err) {
+			createdRoleBinding, err := o.RoleBindingClient.Create(changedRoleBindings[i])
+			if err != nil {
+				return err
+			}
+
+			fmt.Fprintf(o.Out, "clusterrolebinding/%s\n", createdRoleBinding.Name)
+			continue
+		}
+
+		// RoleRef is immutable, to reset this, we have to delete/recreate
+		if !kapi.Semantic.DeepEqual(roleBinding.RoleRef, changedRoleBindings[i].RoleRef) {
+			roleBinding.RoleRef = changedRoleBindings[i].RoleRef
+			roleBinding.Subjects = changedRoleBindings[i].Subjects
+
+			// TODO: for extra credit, determine whether the right to delete/create this rolebinding for the current user came from this rolebinding before deleting it
+			err := o.RoleBindingClient.Delete(roleBinding.Name)
```

Comment:
You should update when possible.  You only need to delete/create when you're changing a roleRef, right?